docs/release-notes/8.16.asciidoc

```diff
@@ -0,0 +1,118 @@
+* To reduce CPU usage, I/O, and event sizes, you can turn on process event aggregation when configuring your {elastic-defend} integration policy. Related process events that occur in rapid succession are combined into fewer aggregate events. To turn on process event aggregation, configure the `advanced.events.aggregate_process` <<adv-policy-settings,advanced policy setting>>.
+* To reduce CPU usage, I/O, and event sizes, you can now turn off of MD5, SHA-1, and SHA-256 hashes in events when configuring your {elastic-defend} integration policy. Example fields include `process.hash.md5` and `file.hash.sha1`. 
+* You can now configure your {elastic-defend} integration policy to allow the collection of SHA-256 file hashes in file events. Before doing so, consider the following caveats: 
+** This can greatly increase {elastic-defend}'s CPU and I/O utilization, and impact system responsiveness.
+** This can significantly delay event enrichment, and lead to Behavioral Protection rules firing too late to effectively stop malicious behavior.
+** This can cause event processing queues to overflow, and lead to dropped events.
+** Many file events won't contain hashes. Hash collection is best effort and not guaranteed to be present in every event. Hashes are collected asynchronously, and shortly after the file activity. Hashes might be missing if the file was rapidly renamed, moved, deleted, or (on Windows) opened by another process without https://learn.microsoft.com/en-us/windows/win32/fileio/creating-and-opening-files[read sharing].
+* Improves {elastic-defend} enabling the use of dynamic {filebeat-ref}/kafka-output.html#topic-option-kafka[topics] for the kafka output.
+* Improves {elastic-defend} by integrating a new Event Tracing for Windows (ETW) provider (Microsoft-Windows-WMI-Activity) to create new event types that can be used by prebuilt endpoint rules to detect malicious WMI activity.
+
+[discrete]
+[[enhancements-8.16.0]]
+==== Enhancements
+* Enables you to open the Rule details flyout from the Alerts table ({kibana-pull}191764[#191764]).
+* Allows you to resize the alert and event details flyouts and choose how it's displayed (over the Alerts table or next to it) ({kibana-pull}192906[#192906], {kibana-pull}182615[#182615]).
+* Allows you to enable entity risk scoring in multiple {kib} spaces ({kibana-pull}192671[#192671]).
+* Creates a new API endpoint for cleaning up entity risk scoring data: `DELETE /api/risk_score/engine/dangerously_delete_data` ({kibana-pull}191843[#191843], {kibana-pull}189872[#189872]). 
+* Allows Automatic Import to analyze a larger number of sample events when generating a new integration ({kibana-pull}196233[#196233]).
+* Improves Automatic Import's ability to recognize events formatted as CSV ({kibana-pull}196228[#196228]).
```

Comment:
```suggestion
* Improves Automatic Import's ability to recognize events formatted as CSV ({kibana-pull}196228[#196228]).
* Automatic Import can now create integrations for logs in the CSV format. Owing to the maturity of log format support, we thus remove the verbiage about requiring the JSON/NDJSON format ({kibana-pull}194386[#194386]).
```

Reply:
Made an edit here to combine both items since they're closely related, hope that works